ed448-goldilocks/src/edwards/affine.rs

```diff
@@ -100,6 +117,34 @@ impl AffinePoint {
         }
     }
 
+    /// Standard compression; store Y and sign of X
+    // XXX: This needs more docs and is `compress` the conventional function name? I think to_bytes/encode is?
```

Comment:
`compress` seems fine to me for this

Reply:
This was copied over as well. Anyway, removed it.